ehrapy/api/encode/__init__.py

```diff
@@ -65,24 +70,41 @@ def undo_encoding(
     return Encoder.undo_encoding(adata, columns, from_cache_file, cache_file)
 
 
-def type_overview(adata: AnnData, sort: bool = False, sort_reversed: bool = False) -> None:
-    """Prints the current state of an AnnData object in a tree format.
+def type_overview(data: Union[MuData, AnnData], sort: bool = False, sort_reversed: bool = False) -> None:
+    """Prints the current state of an :class:`~anndata.AnnData or :class:`~mudata.MuData object in a tree format.
 
     Args:
-        adata: AnnData object to examine
-        sort: Whether the tree output should be sorted
+        data: :class:`~anndata.AnnData or :class:`~mudata.MuData object to display
+        sort: Whether the tree output should be in sorted order
         sort_reversed: Whether to sort in reversed order or not
     """
+    if isinstance(data, AnnData):
+        _adata_type_overview(data, sort, sort_reversed)
+    elif isinstance(data, MuData):
+        _mudata_type_overview(data, sort, sort_reversed)
+    else:
+        print(f"[b red]Unable to present object of type {type(data)}. Can only display AnnData or Mudata objects!")
```

Comment:
Oh, using "b" instead of "bold" now everywhere :) Let's roll with this although I always prefer explicit.
```suggestion
        print(f"[bold red]Unable to present object of type {type(data)}. Can only display AnnData or MuData objects!")
```